docs/setting-toggle.md

```diff
@@ -14,6 +14,7 @@ Inline usage:
   enabled=enabled
   action=(hash
     text="Toggle it!"
+    loading=isToggleSettingRunning
```

Comment:
should we document `disabled` too?

Reply:
Thought about it but I'm not sure there's any real value